packages/agents/sequencer/src/lib/operations/execute/fastpath.ts

```diff
@@ -247,12 +247,8 @@ export const executeFastPathData = async (
   const bidsRoundMap = getBidsRoundMap(bids, config.auctionRoundDepth);
   const availableRoundIds = [...Object.keys(bidsRoundMap)].sort((a, b) => Number(a) - Number(b));
   if ([...Object.keys(bidsRoundMap)].length < 1) {
-    logger.warn("No rounds available for this transferId", requestContext, methodContext, {
-      bidsRoundMap,
-      transferId,
-    });
```

Comment:
IMO, it was useful to check the error status on DD. If we remove logging from here, I guess it would be difficult to figure out the error on DD because we don't have even transferId included in `NoBidsSent`

Reply:
But throwing the error should log it because the caller should log the error. The transferId should be part of requestContext anyways.

Reply:
https://github.com/connext/monorepo/blob/3652-transfer-not-showing-up-as-nobidsreceived-even-though-no-bids-received/packages/agents/sequencer/src/sequencer.ts#L140 right here it will log. This is actually better, the practice should be throw errors in internal functions and try catch at the top level.